shared/AppInsightsCommon/src/Interfaces/ITelemetryContext.ts

```diff
@@ -40,7 +40,7 @@ export interface ITelemetryContext {
     /**
      * The object describing a user tracked by this object.
      */
-    user: IUserContext;
+    readonly user: IUserContext;
```

Comment:
There will be a LOT more than just these.

We need to go over all of the API surface and allocate the correct settings.

eg. All of the TelemetryContext objects should be readonly,